_posts/2022-09-27-call-for-python-software-review-editors.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: single
+title: "pyOpenSci Editorial Process - New Editor in Chief and a Call For Editors"
+excerpt: "pyOpenSci is developing it's open peer review process for Python scientific software. Learn about our structure and if you are interested, apply to be on our editorial board."
+author: "Leah Wasser"
+permalink: /blog/python-software-review-process-editor-in-chief
+header:
+    overlay_color: "#666"
+    overlay_filter: 0.6
+categories:
+  - updates
+  - blog-post
+  - highlight
+  - peer-review
+---
+
+## An update on pyOpenSci's review process 
+
+Hey there! I just wanted to update everyone about where pyOpenSci is with 
+it's peer review process. We are currently hard at work updating [our peer 
+review guides](https://www.pyopensci.org/contributing-guide/intro.html) to 
```

Comment:
suggest changing this to just "our guides" or "our guidebook" to avoid using the phrase "peer review" twice in the same sentence